smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/ConfigurationCleaner.java

```diff
@@ -49,6 +49,7 @@
             "retry-attempts",
             "retry-max-wait",
             "broadcast",
+            "batch",
```

Comment:
Should `"max-queue-size-factor"` be also added? (I don't really know, genuine question :-) )

Reply:
if not kafka client creation will log unrecognised config property